<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>316</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>306</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_numbers">
      <attribute name="title">
       <string>Numbers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Type</string>
         </property>
         <widget class="QRadioButton" name="eightBitRadioButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>23</y>
            <width>44</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>char</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="sixteenBitRadioButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>46</y>
            <width>48</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>short</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="thirtyTwoBitRadioButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>69</y>
            <width>35</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>int</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="sixtyFourBitRadioButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>92</y>
            <width>66</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>long long</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="singleRadioButton">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>23</y>
            <width>347</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>float (single)</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="doubleRadioButton">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>50</y>
            <width>347</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>double</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QCheckBox" name="signedCheckBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Signed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="littleEndianCheckBox">
             <property name="text">
              <string>Little-endian (x86, x86-64)</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="numberEdit">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_dates">
      <attribute name="title">
       <string>Dates</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCalendarWidget" name="calendarWidget"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Hexadecimal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hexEdit">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="changeButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Change value</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
